docs/archive/0.8/sql/functions/nested.md

```diff
@@ -210,38 +210,55 @@ SELECT list_aggr([1, 2, 3], 'string_agg', '-') AS str;
 
 ## Sorting Lists
 
-The function `list_sort` sorts the elements of a list either in ascending or descending order. In addition, it allows to provide whether NULL values should be moved to the beginning or to the end of the list.
```

Comment:
@taniabogatsch this is targeting the archive file in v0.8 of the docs. I'll most it to the latest version.

Reply:
oh oops 😅